data/plugins/linewrapping.lua

```diff
@@ -366,7 +375,9 @@ function DocView:new(doc)
   old_new(self, doc)
   if not open_files[doc] then open_files[doc] = {} end
   table.insert(open_files[doc], self)
-  if config.plugins.linewrapping.enable_by_default then
+  if config.plugins.linewrapping.enable_by_default
+      and matches_any(self.doc.filename or "", config.plugins.linewrapping.files)
```

Comment:
We could use `self.doc.abs_filename` and fallback to `self.doc.filename`.
This way the user could match the entire path, and enable the config only for specific files, files in specific directories, and so on.

Reply:
```suggestion
      and (matches_any(self.doc.filename     or "", config.plugins.linewrapping.files)
      or   matches_any(self.doc.abs_filename or "", config.plugins.linewrapping.files))
```

Something like this?

Reply:
I'd do the `abs` one first, but yeah.

Reply:
```suggestion
      and (matches_any(self.doc.abs_filename or "", config.plugins.linewrapping.files)
      or   matches_any(self.doc.filename     or "", config.plugins.linewrapping.files))
```

Reply:
Maybe also indent the `or` a bit more, so that it's clear that it belongs inside the parenthesis.

Reply:
```suggestion
      and (matches_any(self.doc.abs_filename or "", config.plugins.linewrapping.files)
        or matches_any(self.doc.filename     or "", config.plugins.linewrapping.files))
```